plugins/woocommerce-blocks/tests/e2e/tests/checkout/checkout-block.shopper.block_theme.side_effects.spec.ts

```diff
@@ -583,7 +583,6 @@ test.describe( 'Billing Address Form', () => {
 	const shippingTestData = {
 		firstname: 'John',
 		lastname: 'Doe',
-		company: 'Automattic',
```

Comment:
By default, the company field isn't enabled on the checkout block. I removed it given that it is not critical for this E2E test.